cmd/libs/go2idl/client-gen/generators/generator_for_group.go

```diff
@@ -49,7 +50,11 @@ func (g *genGroup) Namers(c *generator.Context) namer.NameSystems {
 }
 
 func (g *genGroup) Imports(c *generator.Context) (imports []string) {
-	return g.imports.ImportLines()
+	imports = append(imports, g.imports.ImportLines()...)
+	if g.version != "unversioned" {
+		imports = append(imports, "fmt")
```

Comment:
Instead of doing this, you can manually construct an "Errorf" type variable, and then reference it in the template in such a way that the import tracker automatically adds "fmt" as an import. I think that's more robust.